src/main/java/co/rsk/federate/config/PowpegNodeConfigParameter.java

```diff
@@ -5,6 +5,9 @@
 
 public enum PowpegNodeConfigParameter {
   FEDERATOR_ENABLED("federator.enabled", Boolean.toString(true)),
+  // when enabled the federator will be able to attempt signning
+  // rsk transactions waiting for signatures reported by the Bridge,
+  // such as pegouts
```

Comment:
not sure if this line adds value

Reply:
I think it does, maybe in the future we can document what each param does

Reply:
i mean clarifying "such as pegouts", because atm is also enabling migrations for example. If we sign something else in the future we should remember to update the comment. Wdyt?

Reply:
I see, updated